contracts/stellar-upgrader/Cargo.toml

```diff
@@ -11,16 +11,15 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 cfg-if = { workspace = true }
-soroban-sdk = { workspace = true }
 stellar-axelar-std = { workspace = true }
 
 [dev-dependencies]
 dummy-contract = { path = "src/tests/testdata/dummy-contract" }
-soroban-sdk = { workspace = true, features = ["testutils"] }
```

Comment:
can we remove it from the workspace `Cargo.toml` and import it only in `stellar-axelar-std`? Maybe you're planning to do that in a follow-up PR already?